planning/motion_velocity_smoother/launch/motion_velocity_smoother.launch.py

```diff
@@ -77,7 +77,7 @@ def launch_setup(context, *args, **kwargs):
             # Topic for setting maximum speed from the outside (input topic)
             ("~/output/current_velocity_limit_mps",
              "/planning/scenario_planning/current_max_velocity"),
-            ("/localization/kinematic_state", LaunchConfiguration("odom_topic_name"))
+            ("/localization/kinematic_state", LaunchConfiguration("odom_topic"))
```

Comment:
Reason for change?

Reply:
same as https://git.upower.com/group-adas-ad/demo-autonomy/upower_mkz_launch/-/merge_requests/113#note_14794 and https://git.upower.com/group-adas-ad/demo-autonomy/upower_mkz_launch/-/merge_requests/113#note_14796.